src/assets/style.css

```diff
@@ -2,6 +2,14 @@ tr.first-bracket:not(:first-child) > td {
   border-top-width: 5px;
 }
 
+p.descriptionForVariables{
+  font-size: large;
+  margin-top: 3%;
+}
+span.variableMetadataHighlight{
```

Comment:
I chose to change the name to variable-metadata so that is we want to change the look of variable-metadata, we can do it from one place